src/erlcloud_ec2_meta.erl

```diff
@@ -66,6 +69,12 @@ get_instance_dynamic_data(Config) ->
 %%%---------------------------------------------------------------------------
 
 get_instance_dynamic_data(ItemPath, Config) ->
-    DynamicDataPath = "http://169.254.169.254/latest/dynamic/" ++ ItemPath,
+    DynamicDataPath = "http://" ++ host_port() ++ "/latest/dynamic/" ++ ItemPath,
     erlcloud_aws:http_body(erlcloud_httpc:request(DynamicDataPath, get, [], <<>>, erlcloud_aws:get_timeout(Config), Config)).
 
+%%%------------------------------------------------------------------------------
+%%% Internal functions.
+%%%------------------------------------------------------------------------------
+
+host_port() ->
+    application:get_env(erlcloud, metadata_host_port, ?DEFAULT_METADATA_HOST_PORT).
```

Comment:
I would prefer to have this in erlcloud.app.src as default. app.src is the place for all the app env settings. 
there you can also document it via comments . that way both comments will be addressed and good to go.

Reply:
6a337be